.test-defs/CreateShoot.yaml

```diff
@@ -8,6 +8,27 @@ spec:
 
   command: [bash, -c]
   args:
-  - go run -mod=vendor ./.test-defs/cmd/create-shoot
+  - >-
+    go test ./test/integration/shoots/creation
+    -ginkgo.v -ginkgo.progress -ginkgo.noColor
```

Comment:
The default timeout of go test calls is 10 minutes. Therefore the ginkgo timeouts are not respected.
Please add `-timeout=0` to all `go test` calls that we only use the ginkgo defined timeouts